testsuite/MDAnalysisTests/coordinates/test_cif.py

```diff
@@ -0,0 +1,8 @@
+import pytest 
+from MDAnalysisTests.datafiles import PDBX, CIF, MMCIF
+
+
+
+def test_pdbx()
+
+def test_
```

Comment:
Shouldn't be too hard to draft tests based on other coordinate readers and some small test files.